### ISSUE TITLE
Feature request: Pick available port for healthcheck server

### ISSUE THREAD

**Issue by qdm12** [qdm12/ddns-updater#211]
1. What's the feature?

2. Extra information?

**Comment by ansred** [comment]
Any updates on this?

Let me know if you need anything to test in my lab environment

**Comment by qdm12** [comment]
Done in 0c3d258620dcf8e90d71acb5cf2d3e031b235fc9 you can now set `-e HEALTH_SERVER_ADDRESS=127.0.0.1:9998` for example. Didn't try it out of laziness, but it should work! Let me know if it doesn't though 👀

**Comment by ansred** [comment]
The port conflict is gone, but I am getting this error now

`2021/06/08 00:09:31 ERROR cannot write to file "/data/updates.json": mkdir /data: permission denied`


Stack

```
#####################################################################################

version: "3.7"

services: 


  ddns-updater:
    image: qmcgaw/ddns-updater:latest


    container_name: gluetun-Mullvad-ddns-updater
    network_mode: "container:gluetun-Mullvad-VPN"
   #ports:
   #  - 8007:8007/tcp
    volumes:
      - /home/pi/Docker/DDNS-Updater-VPN/data:/updater/data
    environment:
      - CONFIG=
      - PERIOD=5m
      - UPDATE_COOLDOWN_PERIOD=5m
      - PUBLICIP_FETCHERS=all
      - PUBLICIP_HTTP_PROVIDERS=all
      - PUBLICIPV4_HTTP_PROVIDERS=all
      - PUBLICIPV6_HTTP_PROVIDERS=all
      - PUBLICIP_DNS_PROVIDERS=all
      - HTTP_TIMEOUT=10s
      - HEALTH_SERVER_ADDRESS=127.0.0.1:9998

      # Web UI
      - LISTENING_PORT=8007
      - ROOT_URL=/

      # Backup
      - BACKUP_PERIOD=0 # 0 to disable
      - BACKUP_DIRECTORY=/updater/data

      # Other
      - LOG_LEVEL=info
      - LOG_CALLER=hidden
      - GOTIFY_URL=
      - GOTIFY_TOKEN=
    restart: always
    

#####################################################################################
```

Docker Host - Raspberry Pi4 8GB

**Comment by qdm12** [comment]
Step 1 of https://github.com/qdm12/ddns-updater#setup should do I think

**Comment by ansred** [comment]
I did that already and still the issue there.

there is something else in the new image as when I tried this image tag for example it worked fine

`    image: qmcgaw/ddns-updater:v2.2.0`

**Comment by qdm12** [comment]
Indeed, thanks for insisting! I messed it up in my [Maintenance: simplify file paths logic](9b8ec3298b48caa8c271f080962c7fa47feed115) commit this morning 😄  and should be fixed in e19cabc8945b68a4633b48d515857b32adf970b8

**Comment by ansred** [comment]
Things are looking much better now. Testing more in depth the complete stack
ddns
gluetun
webpage server nginx
